public/index.html

```diff
@@ -1,12 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="style.css" rel='stylesheet' type="text/css">
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed&family=Shadows+Into+Light&display=swap" rel="stylesheet">
 </head>
-<body>
 
-  <script src="main.js"></script>
+<body>
+  <div class="main-container">
+    <div class="header">
+      <h1 id="title">Yelp!...ington</h1>
+    </div>
+    <div class="content">
+      <div class="restaurants" id="restaurant-list"></div>
```

Comment:
adding some css to this div to make the `overflow: scroll;` would fix your text from breaking out of the container that it's in.